apertium-uzb-kaa.uzb-kaa.dix

```diff
@@ -19553,6 +19553,5526 @@
+<e>       <p><l>Dovjenko<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Dovjenko<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Doynikov<s n="np"/><s n="cog"/></l>    <r>Doynikov<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Doynikova<s n="np"/><s n="cog"/></l>    <r>Doynikova<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Dragalich<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Dragalich<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Dragich<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Dragich<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Dragichevich<s n="np"/><s n="pat"/></l>    <r>Dragichevich<s n="np"/><s n="pat"/></r></p></e>
+<e>       <p><l>Dragisich<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Dragisich<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Dragomanov<s n="np"/><s n="cog"/></l>    <r>Dragomanov<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Dragomanova<s n="np"/><s n="cog"/></l>    <r>Dragomanova<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Dragovich<s n="np"/><s n="pat"/></l>    <r>Dragovich<s n="np"/><s n="pat"/></r></p></e>
+<e>       <p><l>Dragulesku<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Dragulesku<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Dragutinovich<s n="np"/><s n="pat"/></l>    <r>Dragutinovich<s n="np"/><s n="pat"/></r></p></e>
+<e>       <p><l>Drakulich<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Drakulich<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Draskovich<s n="np"/><s n="pat"/></l>    <r>Draskovich<s n="np"/><s n="pat"/></r></p></e>
+<e>       <p><l>Drazenovich<s n="np"/><s n="pat"/></l>    <r>Drazenovich<s n="np"/><s n="pat"/></r></p></e>
+<e>       <p><l>Drekovich<s n="np"/><s n="pat"/></l>    <r>Drekovich<s n="np"/><s n="pat"/></r></p></e>
+<e>       <p><l>Dresevich<s n="np"/><s n="pat"/></l>    <r>Dresevich<s n="np"/><s n="pat"/></r></p></e>
+<e>       <p><l>Drlyevich<s n="np"/><s n="pat"/></l>    <r>Drlyevich<s n="np"/><s n="pat"/></r></p></e>
+<e>       <p><l>Dropulich<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Dropulich<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Drozdov<s n="np"/><s n="cog"/></l>    <r>Drozdov<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Drozdova<s n="np"/><s n="cog"/></l>    <r>Drozdova<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Druzenko<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Druzenko<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Drzich<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Drzich<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Dubayich<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Dubayich<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Dudakov<s n="np"/><s n="cog"/></l>    <r>Dudakov<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Dudakova<s n="np"/><s n="cog"/></l>    <r>Dudakova<s n="np"/><s n="cog"/></r></p></e>
```

Comment:
You don't want both -ov and -ova forms in the bidix, just the -ov form.  There are quite a few of these that need to be fixed.

Reply:
Still not addressed.